### PR TITLE
Refresh codestart guide for Quarkiverse

### PR DESCRIPTION
**Pull request by ia3andy**

### CODE REVIEW

**Comment by ia3andy** [comment]
@gsmet if you want to create a codestart,maybe it's a good opportunity to also validate this :)

File: docs/src/main/asciidoc/extension-codestart.adoc [hunk 1]
```diff
@@ -71,69 +71,189 @@ The https://github.com/quarkusio/quarkus/tree/main/independent-projects/tools/ba
...
+=== Testing
 
-**You need to build your codestart with Maven to make it available in the tooling:**
+* Add this dependency to the `integration-tests`:
++
+[source,xml]
+----
+<dependency>
+  <groupId>io.quarkus</groupId>
+  <artifactId>quarkus-devtools-testing</artifactId>
```
Comment (ia3andy) [inline docs/src/main/asciidoc/extension-codestart.adoc:181]:
```suggestion
  <artifactId>quarkus-devtools-testing</artifactId>
  <scope>test</scope>
```

**Comment by gsmet** [comment]
I'm having a look today.

**Comment by ia3andy** [comment]
@gsmet I also have this PR ready (I will add a section about it when this gets merged first): https://github.com/quarkusio/quarkus/pull/26806

**Comment by ia3andy** [comment]
ok squashed, just need you stamp @gsmet !

**Comment by ia3andy** [comment]
and thanks for the commit!

**Review by gsmet — APPROVED** [review]
OK, well, given the amount of work I put into it, I would have kept the commit but that's fine :).

**Pull request merged**